packages/frint-store/src/combineEpics.spec.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable import/no-extraneous-dependencies, func-names */
+/* global describe, it */
+import { expect } from 'chai';
+import createStore from './createStore';
+import combineReducers from './combineReducers';
```

Comment:
would really like to see only `combineEpics` function being imported in this spec and unit tested. we don't need to bring in the whole Store functionality for this.

you can take a look at `combineReducers.spec.js` for example, how that function is unit tested.

To test, we just need to make sure, the combined root Epic can trigger all individual Epics, and if they can also stream further actions (if any). 

Can be done with RxJS alone.

Please check if this can be done like this:

```js
import { Subject } from 'rxjs';

import combineEpics from './combineEpics';
import ActionsObservable from './ActionsObservable'; // extract it to a file from Store.js

it('triggers all epics together', function (done) {
  const pingEpic = function (action$) {
    return action$
      .filter(action => action.type === 'PING')
      .map(() => ({ type: 'PONG' });
  };

  const rootEpic = combineEpics(pingEpic);

  const subject$ = new Subject();
  const actions$ = new ActionsObservable(subject);
  const result$ = epic(actions, store);
  const emittedActions = [];

  result.subscribe(emittedAction => emittedActions.push(emittedAction));

  subject.next({ type: 'PING' });

  expect(emittedActions).to.deep.equal([
    // PING
    // PONG
  ]);
});
```